sonic-chassisd/scripts/chassisd

```diff
@@ -73,6 +73,7 @@ CHASSIS_MODULE_REBOOT_INFO_TABLE = 'CHASSIS_MODULE_REBOOT_INFO_TABLE'
 CHASSIS_MODULE_REBOOT_TIMESTAMP_FIELD = 'timestamp'
 CHASSIS_MODULE_REBOOT_REBOOT_FIELD = 'reboot'
 DEFAULT_LINECARD_REBOOT_TIMEOUT = 180
+DEFAULT_DPU_REBOOT_TIMEOUT = 180
```

Comment:
Is 180 secs sufficient for entire DPU reboot time?

Reply:
Also, should we add this to platform.json as vendor configurable timeout?

Reply:
This may not be sufficient for a reboot as I told you.  This is good for shutdown. 
Here is what may work:
shutdown: pre_shutdown_time + graceful shutdown time + 180s for proper HW shutdown (240s is safe)
reboot: shutdown (240s) + hw off-on  requirement if any (say 30s) + 180s for power on

Reply:
A vendor configurable option is a good thing.

Reply:
We can pick a number now and optimize it as we move.

Reply:
@rameshraghupathy during the HDL review sessions we agreed that this value is vendor specific and should be taken from `platform.json`. Please change the implementation based on our agreement

Reply:
Yes, agree, but the existing modular-chassis LC use "platform_env.conf". Shall we be consistent with that?
PLATFORM_ENV_CONF_FILE = "/usr/share/sonic/platform/platform_env.conf"
        self.linecard_reboot_timeout = DEFAULT_LINECARD_REBOOT_TIMEOUT
        if os.path.isfile(PLATFORM_ENV_CONF_FILE):
            with open(PLATFORM_ENV_CONF_FILE, 'r') as file:
                for line in file:
                    field = line.split('=')[0].strip()
                    if field == "linecard_reboot_timeout":
                        self.linecard_reboot_timeout = int(line.split('=')[1].strip())